MockKit/MockService.swift

```diff
@@ -25,25 +25,24 @@ public final class MockService: Service {
     
     public let maxHistoryItems = 1000
     
-    public var history: [String]
-    private let historyQueue = DispatchQueue(label: "MockService_historyQueue")
+    private var _history = Locked<[String]>([])
```

Comment:
[nit] the pattern we've been using is to name this variable `lockedHistory`, rather than have an underscore.